logger/logger_writer.go

```diff
@@ -146,6 +147,8 @@ func queueMsg(lvl Level, prefix, format string, v ...interface{}) (err error) {
 		return
 	}
 
+	msg.time = time.Now()
+
```

Comment:
Okay -- I'll go ahead and clean that up on this same branch, and resubmit on Monday.